crispy-api/api/__init__.py

```diff
@@ -29,10 +29,9 @@ def init_app(debug: bool) -> FastAPI:
 
 
 @app.on_event("startup")
-async def init_database(MONGODB_NAME: Optional[str] = None) -> None:
-    mongo_thingy.connect(
-        MONGO_URI, database_name=MONGODB_NAME, uuidRepresentation="standard"
-    )
+def init_database() -> None:
+    set_storage("./data", storage="sqlite")
+    connect("./data", client_cls=MontyClient)
```

Comment:
Also, right now, starting tests will clear the `./data`
We don't want that.
```suggestion
def init_database(path: Optional[str] = DATABASE_PATH) -> None:
    set_storage(path, storage="sqlite")
    connect(path, client_cls=MontyClient)
```

And in the tests, connect with `./tests-data`

Don't forge to gitignore them.